{{cookiecutter.project_slug}}/.env.example

```diff
@@ -141,3 +141,9 @@ SMTP_HOST='smtp.mailgun.org'
 SMTP_PORT='587'
 SMTP_VALID_TESTING_DOMAINS='thinknimble.com'
 {% endif %}
+
+# Testing (NOTE: Heroku and Github Actions will need to have matching values for some of these)
+DJANGO_SUPERUSER_PASSWORD='!!!DJANGO_SECRET_KEY!!!'
+CYPRESS_baseUrl='http://localhost:8080'
+CYPRESS_TEST_USER_PASS='!!!DJANGO_SECRET_KEY!!!'
+CYPRESS_TEST_USER_EMAIL='cypress@example.com'
```

Comment:
running `db_setup.sh` will inject this user with these creds into your DB so this will work.
We don't have the project specific app.json yet, but when we do it's easy to have this also run automatically for all review apps.